bookdown/01-intro.Rmd

```diff
@@ -1,75 +1,75 @@
 # Introduction and Overview
 
-The `r mlr_pkg("mlr3")` [@mlr3] package and [ecosystem](https://github.com/mlr-org/mlr3/wiki/Extension-Packages) provide a generic, object-oriented, and extensible framework for [classification](#tasks), [regression](#tasks), [survival analysis](#survival), and other machine learning tasks for the R language [@R].
+The `mlr3` [@mlr3] package and [ecosystem](https://github.com/mlr-org/mlr3/wiki/Extension-Packages) provide a generic, object-oriented, and extensible framework for [classification](#tasks), [regression](#tasks), [survival analysis](#survival), and other machine learning tasks for the R language [@R].
```

Comment:
1. This removed the link to mlr3
2. Why are some hyperrefs now formated as footnotes? What is wrong with being able to directly click stuff?

Reply:
1. We can think about keeping the `mlr_pkg()` and `*_pkg()` calls in general. Maybe removing those was not needed since those notations do not get hyperlinked by the auto-linker.

1. You mean in the PDF version? This was the case ever since? Some pages had a footnote list that was spanning the half height of the page due to the amount of link.

Reply:
Before, all links in the PDF were formatted as footnotes because there was a renewcommand in the LaTeX preamble. I cleared that.